qiskit/execute_function.py

```diff
@@ -104,7 +123,7 @@ def execute(
                ``[[0, 1], [0, 3], [1, 2], [1, 5], [2, 5], [4, 1], [5, 3]]``
 
         backend_properties (BackendProperties):
-            Properties returned by a backend, including information on gate
+            DEPRECATED: Properties returned by a backend, including information on gate
             errors, readout errors, qubit coherence times, etc. Find a backend
             that provides this information with:
             ``backend.properties()``
```

Comment:
Below this line, how come `initial_layout` got to stay non-deprecated?  That feels like a transpiler option as well, perhaps?

Reply:
I was on the fence about it. It felt like maybe high level enough to say run this circuit on these qubits. But, I agree it's probably better to deprecate it here too and rely on `transpile()` if you need this.